notebooks/llm-chatbot/llm-chatbot.ipynb

```diff
@@ -296,7 +296,7 @@
     "The command bellow demonstrates basic command for model export with `optimum-cli`\n",
```

Comment:
<p>can we also add reference on rag notebook in this notebook? Something like "next steps"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
you mean in the end of llm-chatbot ?